tests/framework/extensions/namespaces/namespaces.go

```diff
@@ -142,3 +141,43 @@ func CreateNamespace(client *rancher.Client, namespaceName, containerDefaultReso
 	}
 	return newNamespace, nil
 }
+
+
+// ListNamespace is a helper function that uses the dynamic client to get a list of all namespaces from a cluster from all projects.
+func ListNamespace(client *rancher.Client, clusterID string) (*kubeUnstructured.UnstructuredList, error) {
```

Comment:
Hey Anu Ugur has a PR with this helper, which I would like to be merged and used instead. 
https://github.com/rancher/rancher/pull/38171